contracts/levels/AlienCodex.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.5.0;
 
-import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
+import '../helpers/Ownable-05.sol';
```

Comment:
We can install a previous version of OZ Contracts under a different name:

```
npm install -D openzeppelin-contracts-2.5@npm:@openzeppelin/contracts@2.5.0
```

Later:

```
import 'openzeppelin-contracts-2.5/ownership/Ownable.sol';
```

Do you think this would be better than vendoring the `Ownable` contract?

Reply:
Woah I didnt know that. The Ownable contract was also breaking the game because it emits an event in the constructor - do you know if there was an 0.5 version that didnt do that? The game definitely used to work so I'm wondering how that was

Reply:
Ownable emits an event in the constructor since the contracts were for solidity 0.4.

So wait is the level working as intended?

Reply:
The level works with this `Ownable-05` because I removed the event from the constructor. I could use the official OZ 05 Ownable and figure out a work around into the front end if you prefer?